packages/valory/skills/termination_abci/tests/test_rounds.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2021-2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Test the rounds of the skill."""
+
+from copy import deepcopy
+from typing import FrozenSet, cast
+
+from packages.valory.skills.abstract_round_abci.base import AbciAppDB, ConsensusParams
+from packages.valory.skills.termination_abci.payloads import BackgroundPayload
+from packages.valory.skills.termination_abci.rounds import (
+    BackgroundRound,
+    Event,
+    SynchronizedData,
+    TerminationRound,
+)
+
+
+MAX_PARTICIPANTS: int = 4
+
+
+def get_participants() -> FrozenSet[str]:
+    """Participants"""
+    return frozenset([f"agent_{i}" for i in range(MAX_PARTICIPANTS)])
```

Comment:
why is this a separate function?

Reply:
It's a pattern we have all over the round tests. I assume its like this because it doesnt have any logic related to the test class, ie no `self` or `cls` use.